### PR TITLE
Minor change PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -134,7 +134,7 @@
     "notice_1": "Somente você controla e tem acesso ao seus ativos, qualquer ação que você realizar pode causar uma perda potencial de fundos.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:134]:
```suggestion
    "notice_1": "Somente você controla e tem acesso ao seus ativos, qualquer ação que você realizar pode causar uma perda substancial de fundos.",
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -131,10 +131,10 @@
   },
   "footer": {
     "title": "Distribuído por <0>Bitcoin</0>",
-    "notice_1": "Somente você controla e tem acesso ao seus ativos, qualquer ação que você realizar pode causar uma perda potencial de fundos.",
+    "notice_1": "Somente você controla e tem acesso ao seus ativos, qualquer ação que você realizar pode causar uma perda substancial de fundos.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:134]:
```suggestion
    "notice_1": "Somente você controla e tem acesso ao seus ativos, qualquer ação que você realizar pode causar uma perda considerável de fundos.",
```

**Comment by bitcoinuser** [comment]
@creed-victor You can merge now. Thanks.

**Review by creed-victor — APPROVED** [review]

**Comment by creed-victor** [comment]
@bitcoinuser you can also create PR's as draft in the future If you know that you will need more changes or want to check on preview build first :)

**Pull request merged**